client-js/app/elements/labrad-plot.html

```diff
@@ -26,6 +33,7 @@
   </style>
   <template>
     <div id="plot"></div>
+    <div id="pos">{{currPos}}</div>
```

Comment:
It's a bit strange to me for ```currPos``` to also include the text ````current position:```

Probably splitting hairs here but the way that seems natural to me is to write this as:

```
<div id="pos">The current position is: <span>{{currPos}}</span></div>
```